src/ui/styles/colors.scss

```diff
@@ -62,6 +62,8 @@
     #a8c55f 123.53%,
     #b9cf7c 165.71%
   );
+
+  --black: #000;
```

Comment:
I think you misunderstood my comment here:

https://github.com/cardano-foundation/cf-identity-wallet/pull/384#discussion_r1537325750

No need to define `black`. Please roll back this change.